react/features/base/participants/components/Avatar.native.js

```diff
@@ -289,13 +279,36 @@ export default class Avatar extends Component<Props, State> {
      * @inheritdoc
      */
     render() {
-        const { source, useDefaultAvatar } = this.state;
+        const { source } = this.state;
 
         return (
             <Fragment>
                 { source && this._renderAvatar() }
-                { useDefaultAvatar && this._renderDefaultAvatar() }
+                { this.useDefaultAvatar && this._renderDefaultAvatar() }
             </Fragment>
         );
     }
 }
+
+/* eslint-disable react/no-multi-comp */
+
+/**
+ * Implements an avatar as a React Native/mobile {@link Component}.
+ *
+ * Note: we use `key` in order to trigger a new component creatin in case
+ * the URI changes.
```

Comment:
Are avatar uris guaranteed to be unique?

Reply:
Nope, but why would that matter? There will never be 2 Avatar components hanging off the same parent with matching keys.

Reply:
Okay, then it doesn't matter. I think the use of key to force a re-initialization is something I'm not used to so it's initially confusing to see, so I was thinking of the unique-key requirements. But in current usage it won't be a problem.

Reply:
Reading more about key it wouldn't be a problem period so I guess that's that.